sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/MetricDataMapper.java

```diff
@@ -151,7 +152,17 @@ public static void updateMetricPointBuilder(
         pointBuilder.setValue(pointDataValue);
         // TODO (heya) why give it the same name as otel metric?
         //  it seems this field doesn't matter and only _MS.MetricId property matters?
-        pointBuilder.setName(metricData.getName());
+
+        // We emit jmx metrics via opentelemetry with an attribute that has the original name the customer set
+        // in their applicationinsights.json file. We parse for that attribute here to report the intended
+        // metric name to Breeze.
+        String jmxMetricName = metricData.getResource().getAttribute(AttributeKey.stringKey("app_insights_internal_jmx_metric"));
```

Comment:
I think we want to check for metric attribute as opposed to a resource attribute